docs/README.Windows.md

```diff
@@ -77,17 +77,17 @@ Default options are fine.
 Start the VS2017 installer and click `Individual components`.
 * Under **Compilers, build tools and runtimes** select
   * `Msbuild`
-  * `VC++ 2017 v141 toolset (x86,x64)`
+  * `VC++ 2017 version 15.x v14.x latest v141 tools`
   * `Visual C++ 2017 Redistributable Update`
-  * `Visual C++ compilers and libraries for ARM`
-  * `Visual C++ compilers and libraries for ARM64`
-  * `Visual C++ runtime for UWP`
+  * `Visual C++ compilers and libraries for ARM` (if compiling for ARM)
```

Comment:
There used to be a bug where visual studio complained and refused to open the projects unless arm tooling was installed.

Have you tested without it installed without getting any errors when loading a x86/x64 solution?

Reply:
Ok I checked this and it's still an issue for UWP. the UWP "Workload" includes arm and arm64 compilers and if they're not installed Visual Studio refuses to load the projects.
```suggestion
  * `Visual C++ compilers and libraries for ARM` (if compiling for ARM or UWP)
```